components/backup/src/endpoint.rs

```diff
@@ -716,13 +746,8 @@ impl<E: Engine, R: RegionInfoProvider + Clone + 'static> Endpoint<E, R> {
             },
         };
 
-        let limit = self.softlimit.limit();
-        self.pool.borrow_mut().spawn(move || {
-            tikv_alloc::add_thread_memory_accessor();
+        self.pool.borrow_mut().spawn(async move {
             let _with_io_type = WithIOType::new(IOType::Export);
-            defer!({
-                tikv_alloc::remove_thread_memory_accessor();
-            });
 
             // Check if we can open external storage.
             let backend = match create_storage(&request.backend, config) {
```

Comment:
this line can move outside.

Reply:
Yes, but maybe let's done it in another PR?